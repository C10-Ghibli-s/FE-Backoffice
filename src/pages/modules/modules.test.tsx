import Modules from '../modules';
import renderer from 'react-test-renderer';

it('should render modules card correctly', () => {
  const tree = renderer
    .create(
      <Modules>
        <h1>Modules</h1>
        <section></section>
      </Modules>
      )
    .toJSON();
  expect(tree).toMatchSnapshot();
})