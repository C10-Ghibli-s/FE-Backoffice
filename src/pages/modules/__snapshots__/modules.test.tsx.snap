// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render modules card correctly 1`] = `
Array [
  <h1
    className="p-2 mx-8 mt-5 text-3xl font-semibold text-gray-900 border-l-4 border-blue-500 sm:text-left sm:ml-32"
  >
    Modules
  </h1>,
  <section
    className="flex flex-col items-center justify-center my-8"
  >
    <section
      className="relative flex flex-col sm:flex sm:flex-row w-5/6 sm:h-28 sm:justify-between p-2 mt-1 align-middle sm:items-center bg-white rounded-xl hover:cursor-pointer hover:outline outline-4 outline-blue-400 "
      onClick={[Function]}
    >
      <span
        className="absolute top-4 w-8 h-8 sm:h-auto mx-3 bg-center bg-no-repeat bg-cover bg-users-icon"
      />
      <h2
        className="p-2 text-2xl text-gray-900 sm:h-fit indent-10"
      >
        Users
      </h2>
      <p
        className="p-2 m-2 italic text-gray-400 sm:pr-8 sm:h-fit right-12"
      >
        Find , create, edit or delete users
      </p>
      <span
        className="-rotate-90 transition bg-center bg-no-repeat bg-contain h-4 w-4 absolute right-2 bottom-14 sm:bottom-12 bg-[url('../assets/icons/arrow-icon.svg')]"
      />
    </section>
    <section
      className="relative flex flex-col sm:flex sm:flex-row w-5/6 sm:h-28 sm:justify-between p-2 mt-1 align-middle sm:items-center bg-white rounded-xl hover:cursor-pointer hover:outline outline-4 outline-blue-400 "
      onClick={[Function]}
    >
      <span
        className="absolute top-4 w-8 h-8 sm:h-auto mx-3 bg-center bg-no-repeat bg-cover bg-movies-icon"
      />
      <h2
        className="p-2 text-2xl text-gray-900 sm:h-fit indent-10"
      >
        Movies
      </h2>
      <p
        className="p-2 m-2 italic text-gray-400 sm:pr-8 sm:h-fit right-12"
      >
        Release a new movie, edit, or delete it
      </p>
      <span
        className="-rotate-90 transition bg-center bg-no-repeat bg-contain h-4 w-4 absolute right-2 bottom-14 sm:bottom-12 bg-[url('../assets/icons/arrow-icon.svg')]"
      />
    </section>
    <section
      className="relative flex flex-col sm:flex sm:flex-row w-5/6 sm:h-28 sm:justify-between p-2 mt-1 align-middle sm:items-center bg-white rounded-xl hover:cursor-pointer hover:outline outline-4 outline-blue-400 "
      onClick={[Function]}
    >
      <span
        className="absolute top-4 w-8 h-8 sm:h-auto mx-3 bg-center bg-no-repeat bg-cover bg-writers-icon"
      />
      <h2
        className="p-2 text-2xl text-gray-900 sm:h-fit indent-10"
      >
        Writers
      </h2>
      <p
        className="p-2 m-2 italic text-gray-400 sm:pr-8 sm:h-fit right-12"
      >
        Add or edit a writer
      </p>
      <span
        className="-rotate-90 transition bg-center bg-no-repeat bg-contain h-4 w-4 absolute right-2 bottom-14 sm:bottom-12 bg-[url('../assets/icons/arrow-icon.svg')]"
      />
    </section>
    <section
      className="relative flex flex-col sm:flex sm:flex-row w-5/6 sm:h-28 sm:justify-between p-2 mt-1 align-middle sm:items-center bg-white rounded-xl hover:cursor-pointer hover:outline outline-4 outline-blue-400 "
      onClick={[Function]}
    >
      <span
        className="absolute top-4 w-8 h-8 sm:h-auto mx-3 bg-center bg-no-repeat bg-cover bg-directors-icon"
      />
      <h2
        className="p-2 text-2xl text-gray-900 sm:h-fit indent-10"
      >
        Directors
      </h2>
      <p
        className="p-2 m-2 italic text-gray-400 sm:pr-8 sm:h-fit right-12"
      >
        Add or edit a director
      </p>
      <span
        className="-rotate-90 transition bg-center bg-no-repeat bg-contain h-4 w-4 absolute right-2 bottom-14 sm:bottom-12 bg-[url('../assets/icons/arrow-icon.svg')]"
      />
    </section>
    <section
      className="relative flex flex-col sm:flex sm:flex-row w-5/6 sm:h-28 sm:justify-between p-2 mt-1 align-middle sm:items-center bg-white rounded-xl hover:cursor-pointer hover:outline outline-4 outline-blue-400 "
      onClick={[Function]}
    >
      <span
        className="absolute top-4 w-8 h-8 sm:h-auto mx-3 bg-center bg-no-repeat bg-cover bg-musicians-icon"
      />
      <h2
        className="p-2 text-2xl text-gray-900 sm:h-fit indent-10"
      >
        Musicians
      </h2>
      <p
        className="p-2 m-2 italic text-gray-400 sm:pr-8 sm:h-fit right-12"
      >
        Add or edit a musician
      </p>
      <span
        className="-rotate-90 transition bg-center bg-no-repeat bg-contain h-4 w-4 absolute right-2 bottom-14 sm:bottom-12 bg-[url('../assets/icons/arrow-icon.svg')]"
      />
    </section>
    <section
      className="relative flex flex-col sm:flex sm:flex-row w-5/6 sm:h-28 sm:justify-between p-2 mt-1 align-middle sm:items-center bg-white rounded-xl hover:cursor-pointer hover:outline outline-4 outline-blue-400 "
      onClick={[Function]}
    >
      <span
        className="absolute top-4 w-8 h-8 sm:h-auto mx-3 bg-center bg-no-repeat bg-cover bg-roles-icon"
      />
      <h2
        className="p-2 text-2xl text-gray-900 sm:h-fit indent-10"
      >
        Roles
      </h2>
      <p
        className="p-2 m-2 italic text-gray-400 sm:pr-8 sm:h-fit right-12"
      >
        Assign new roles
      </p>
      <span
        className="-rotate-90 transition bg-center bg-no-repeat bg-contain h-4 w-4 absolute right-2 bottom-14 sm:bottom-12 bg-[url('../assets/icons/arrow-icon.svg')]"
      />
    </section>
  </section>,
]
`;
